src/components/Button/CozyHome.jsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { translate } from 'cozy-ui/react/I18n'
+import { ButtonLink } from 'cozy-ui/react'
+
+const CozyHome = ({ t, from }) => (
+  <ButtonLink
+    label={t('Share.create-cozy')}
+    href={`https://cozy.io/try-it?from=${encodeURIComponent(from)}`}
```

Comment:
There is some convention to detect sources in links, like __utc__ query parameters, no?

Reply:
Sorry, I don't understand what you mean?